sonar-dotnet-shared-library/src/test/java/org/sonarsource/dotnet/shared/plugins/GeneratedFileFilterTest.java

```diff
@@ -56,44 +55,32 @@ public void setUp() {
       SonarRuntimeImpl.forSonarQube(Version.create(7, 9), SonarQubeSide.SERVER, SonarEdition.COMMUNITY)) {
     };
     MapSettings settings = new MapSettings(new PropertyDefinitions(definitions.create()));
-    defaultConfiguration = new AbstractSolutionConfiguration(settings.asConfig(), "cs") { };
+    defaultConfiguration = new AbstractSolutionConfiguration(settings.asConfig(), "cs") {
+    };
   }
 
   @Test
   public void accept_returns_false_for_autogenerated_files() throws IOException {
     // Arrange
-    GeneratedFileFilter filter = createFilter("autogenerated", defaultConfiguration);
+    InputFile inputFile = mockInputFile("autogenerated");
+    GeneratedFileFilter filter = createFilter(inputFile, defaultConfiguration);
 
     // Act
-    Boolean result = filter.accept(newInputFile("autogenerated"));
+    Boolean result = filter.accept(inputFile);
 
     // Assert
     assertThat(result).isFalse();
     assertThat(logs.logs(LoggerLevel.DEBUG)).contains("Will ignore generated code");
     assertThat(logs.logs(LoggerLevel.DEBUG)).contains("Skipping auto generated file: autogenerated");
   }
 
-  @Test
-  public void accept_is_not_case_sensitive() throws IOException {

```

Comment:
why remove this?

Reply:
It's no longer responsibility of GeneratedFileFilter to take care about case sensitivity. And it has lost it's value - take a look at the current mock 
```
when(processor.isGenerated(generatedInputFile)).thenReturn(true);
```
It doesn't really test anything related to case sensitivity. So I removed it to deobfuscate this fact.